pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -444,6 +448,26 @@ func (r *ReconcileInfinispan) Reconcile(request reconcile.Request) (reconcile.Re
 	return reconcile.Result{}, err
 }
 
+func configureLoggers(pods *corev1.PodList, cluster ispn.ClusterInterface, infinispan *infinispanv1.Infinispan) error {
+	if infinispan.Spec.Logging != nil && len(infinispan.Spec.Logging.Categories) > 0 {
```

Comment:
We can return early to avoid additional nesting:

```
	if infinispan.Spec.Logging == nil || len(infinispan.Spec.Logging.Categories) == 0 {
		return nil
	}
```

Reply:
Yes, sure, will fix